frontend/src/scenes/session-recordings/player/inspector/v2/PlayerInspectorControls.tsx

```diff
@@ -53,7 +73,15 @@ export function PlayerInspectorControls({
                             key={tabId}
                             size="small"
                             // We want to indicate the tab is loading, but not disable it so we just override the icon here
-                            icon={tabsState[tabId] === 'loading' ? <Spinner monocolor /> : TabToIcon[tabId]}
+                            icon={
+                                tabId !== SessionRecordingPlayerTab.ALL ? (
+                                    tabsState[tabId] === 'loading' ? (
+                                        <Spinner monocolor />
+                                    ) : (
+                                        cloneElement(tabToIconAndDescription[tabId].icon, { width: 24, height: 24 })
```

Comment:
We could perhaps do this better by changing the icon to be a class rather than a rendered JSX so we can just render it ourselves without needing to cloneElement